x/auth/types/amino_signing_test.go

```diff
@@ -46,9 +46,9 @@ func TestLegacyAminoJSONHandler_GetSignBytes(t *testing.T) {
 
 	handler := stdTxSignModeHandler{}
 	signingData := signing.SignerData{
-		ChainID:         chainId,
-		AccountNumber:   accNum,
-		AccountSequence: seqNum,
+		ChainID:       chainId,
+		AccountNumber: accNum,
```

Comment:
TBH, I'd prefer consistency here. Can we stick with Account prefix or omit it entirely from both fields?

Reply:
I really don't have a strong preference. cc @webmaster128

Reply:
My impression was that it is called "account number" and "sequence" in the public interfaces, so this is what I am used to. Turns out the current state is more inconsistent than that.

I don't see a lot of value in having the "account" prefix in general. To be consistent, you'd also have to call it "account pubkey", "account balance" and so on for all the cases where account is implicitly clear.

For "number" I don't think it can be dropped as number can be anything.

Reply:
I'd opt for `Account.AccountSequence` & `Account.AccountNumber` or drop the prefix entirely for both. Number does mean something when its a field under `Account`.

Reply:
@alexanderbez 

I will not disagree with you on the naming issue, but this has been done all over the place in the SDK and that should be addressed in another issue, not just here. The main public object we see is [Account](https://github.com/cosmos/cosmos-sdk/blob/master/x/auth/types/account.go#L29-L30) which has `AccountNumber` and `Sequence`. 

This is also defined in [`AccountI`](https://github.com/cosmos/cosmos-sdk/blob/master/x/auth/types/account.go#L305-L309)

The Signature info should be consistent with that. I would propose this PR to be merged following the same naming as `Account`. And then you can make an issue to change all `AccountNumber` to `Number`, which could be another PR as clearly out of scope here. There may also be some debate on that point and all the other core sdk members should have a chance to weigh in